src/APIFunctions/S3Bucket.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import { MAILER_API_URL } from '../config/config.example.json';
+
+export function uploadToS3(files) {
+  for (let i = 0; i < files.length; i++) {
+    let file = files[i];
+    let fileParts = req.body.file.split('.');
+    let fileName = fileParts[0];
+    let fileType = fileParts[1];
+
+    // Retrieving signed url which will allow us to make an upload request to S3
+    axios
+      .post(MAILER_API_URL + '/api/S3Bucket/getSignedUrl', {
+        fileName: fileName,
+        fileType: fileType
```

Comment:
remember this in the future for javascript
```suggestion
        fileName,
        fileType
```